std/datetime.d

```diff
@@ -1075,6 +1075,42 @@ public:
         //assert(ist.opCmp(ist) == 0);
     }
 
+    /**
+     * Returns: A hash of the $(LREF SysTime)
+     */
+    size_t toHash() const @nogc pure nothrow @safe
+    {
+        static if (is(size_t == ulong))
+        {
+            return _stdTime;
+        }
+        else
+        {
+            enum uint m = 0x5bd1e995;
```

Comment:
This is the 32bit version, isn't that the one this should be using since we're looking to get a `uint`?